version_info.json

```diff
@@ -1,10 +1,9 @@
 {
   "pinned-versions": {
-    "iree-compiler": "20220809.230",
-    "iree-runtime": "20220809.230",
-    "iree-tools-xla": "20220809.230",
-    "jax": "0.3.15",
+    "iree-compiler": "20220814.235",
```

Comment:
We're looking to release 20220811.232. Is it possible to use that instead?

Reply:
Yup, piece of cake. Updated to use the stable release.